app/models/concerns/metric_types.rb

```diff
@@ -18,6 +18,10 @@ module MetricTypes
     ],
     'subject.first-to-classify' => [
       'classification_created'
+     ],
+    'classifier.first-to-classify.visible' => [
```

Comment:
Looking at that variant key list again, I suppose it could be `subject.first-to-classify.visible`. `subject.first-to-classify` is a text split that's in the classification summary.